volume/stat.go

```diff
@@ -0,0 +1,138 @@
+package volume
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/puppetlabs/wash/munge"
+	"github.com/puppetlabs/wash/plugin"
+)
+
+// StatCmd returns the command required to stat all the files in a directory.
+func StatCmd(path string) []string {
+	// size, atime, mtime, ctime, mode, name
+	// %s - Total size, in bytes
+	// %X - Time of last access as seconds since Epoch
+	// %Y - Time of last data modification as seconds since Epoch
+	// %Z - Time of last status change as seconds since Epoch
+	// %f - Raw mode in hex
+	// %n - File name
+	return []string{"find", path, "-mindepth", "1", "-exec", "stat", "-c", "%s %X %Y %Z %f %n", "{}", "+"}
+}
+
+// Keep as its own specialized function as it will be faster than munge.ToTime.
+func parseTime(t string) (time.Time, error) {
+	epoch, err := strconv.ParseInt(t, 10, 64)
+	if err != nil {
+		return time.Time{}, err
+	}
+	return time.Unix(epoch, 0), nil
+}
+
+// StatParse parses a single line of the output of StatCmd into EntryAttributes and a name.
```

Comment:
I'd change `name` => `path` since that's what it is returning. Basing that off of `StatParseAll`'s code, specifically the `fullpath` reference.